learning-resources/c++ Links

```diff
@@ -0,0 +1,12 @@
+(VIDEOS)
+
+(1) https://www.youtube.com/watch?v=vLnPwxZdW4Y
+(2) https://www.youtube.com/watch?v=MhYECGUzdA4
```

Comment:
Please do not use bare URLs, instead use the standard markdown format, for example, [name of video, from channel-name](link)